code/modules/cargo/packs.dm

```diff
@@ -990,7 +990,29 @@
 					/obj/item/pizzabox/meat,
 					/obj/item/pizzabox/vegetable)
 	crate_name = "pizza crate"
-	
+	var/static/anomalous_box_provided = FALSE
+
+/datum/supply_pack/organic/pizza/fill(obj/structure/closet/crate/C)
+	. = ..()
+	if(!anomalous_box_provided)
+		for(var/obj/item/pizzabox/P in C)
+			if(prob(1)) //1% chance for each box, so 4% total chance per order
+				var/obj/item/pizzabox/infinite/fourfiveeight = new(C)
+				fourfiveeight.boxtag = P.boxtag
+				qdel(P)
+				anomalous_box_provided = TRUE
+				message_admins("An anomalous pizza box was provided in a pizza crate!")
```

Comment:
are you serious

Reply:
yes, I figured it was noteworthy enough to ping because it's an object that provides infinite amounts of food to the entire station

Reply:
we can see the event message like everyone else, it doesn't seem necessary

Reply:
The message only has a 50% chance to trigger; I just made clearer for admins in case it didn't. I can make it a guaranteed chance to send the announce if you'd like, and remove this.

Reply:
or just message them if the report isn't sent